src/queue-list/laboratory-patient-list.component.tsx

```diff
@@ -47,6 +47,16 @@ interface RejectOrderProps {
 const LaboratoryPatientList: React.FC<LaboratoryPatientListProps> = () => {
   const { t } = useTranslation();
 
+  const OrderStatuses = [
+    t("all", "All"),
+    t("received", "RECEIVED"),
+    t("inProgressOrder", "IN_PROGRESS"),
+    t("completedOrder", "COMPLETED"),
+    t("exception", "EXCEPTION"),
+    t("on_hold", "ON_HOLD"),
+    t("decline", "DECLINED"),
+  ];
```

Comment:
Hi @gitcliff i think this might have been of a mixture of approach for this. The values here are referenced in the backend so the translations isn't something that applies here otherwise someone can fix a typo thinking these texts don't have an impact but in reality the values have to stay intact. 
One approach we could use is to map plain text to the values passed but for now let's just remove the translations so that this doesn't drag.  
```suggestion
  const OrderStatuses = [
    "All",
    "RECEIVED",
    "IN_PROGRESS",
    "COMPLETED",
    "EXCEPTION",
    "ON_HOLD",
    "DECLINED",
  ];
```

Reply:
> Hi @gitcliff i think this might have been of a mixture of approach for this. The values here are referenced in the backend so the translations isn't something that applies here otherwise someone can fix a typo thinking these texts don't have an impact but in reality the values have to stay intact. One approach we could use is to map plain text to the values passed but for now let's just remove the translations so that this doesn't drag.

thanks @pirupius ,,i have updated